# Generated by Django 4.2.2 on 2023-07-24 20:28

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0026_alter_review_user_review_first_last_name_unique'),
    ]

    operations = [
        migrations.RemoveConstraint(
            model_name='review',
            name='first_last_name_unique',
        ),
        migrations.AddConstraint(
            model_name='review',
            constraint=models.UniqueConstraint(models.F('facility'), models.F('user'), name='first_last_name_unique', violation_error_message='You have already submitted a review for this facility'),
        ),
    ]
